tests/plugins/functions/text/output_data/failed_with_severity.stdout

```diff
@@ -13,7 +13,9 @@ Hello from CRITICAL
 [1m[32m^^^^ END read_data ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 [1m[34m* dev4.group_2 ** changed : False **********************************************
 [1m[31mvvvv read_data ** changed : False vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv ERROR
-Exception('Unknown Error -> Contact your system administrator',)
+BrigadeSubTaskError()
```

Comment:
Shouldn't this have been something like `Subtask: parse_data (failed)\n`?

Reply:
It's due to this:
https://github.com/brigade-automation/brigade/blob/develop/tests/plugins/functions/text/test_print_result.py#L101

Because the result key isn't printed. That's to avoid showing the traceback which contains my local paths.

Reply:
I think that should output the task name and the arguments used if possible. If it's too hard we can open an issue and fix it later